### PR TITLE
Docs: update troubleshooting doc

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
 ## Container keeps pending/ terminating
 
 1. Check the output of `kubectl get nodes`
-2. Check the output of `kubectl describe pod <your-pod>`
```
Comment (fisherxu) [inline docs/troubleshooting.md:8]: `kubectl describe pod <your-pod>` can help check whether the pod is scheduled successfully.

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
...
-3. Check the `edgecore` logs for any errors.
-4. Check the architecture of the node running `edgecore` and make sure that container image you are trying to run is of the same architecture.
+2. Check the `edgecore` logs for any errors.
+3. Check the architecture of the node running `edgecore` and make sure that container image you are trying to run is of the same architecture.
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:9]:
```suggestion
2. Use `kubectl describe pod <your-pod>` to check whether the pod is scheduled successfully.
3. Check the `edgecore` logs for any errors.
4. Check the architecture of the node running `edgecore` and make sure that container image you are trying to run is of the same architecture.
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
...
    For example, if you are running `edgecore` on Raspberry Pi 4, which is of `arm64v8` architecture, the nginx image to be executed would be `arm64v8/nginx` from the docker hub.
 
-5. Also, check that the `podSandboxImage` is correctly set as defined in [Modification in edgecore.yaml](./configuration/kubeedge.md#modification-in-edgecoreyaml).
+4. Also, check that the `podSandboxImage` is correctly set as defined in [Modification in edgecore.yaml](./configuration/kubeedge.md#modification-in-edgecoreyaml).
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:12]:
```suggestion
5. Also, check that the `podSandboxImage` is correctly set as defined in [Modification in edgecore.yaml](./configuration/kubeedge.md#modification-in-edgecoreyaml).
```

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
...
 
-6. If all of the above is correctly set, login manually to your edge node and run your docker image manually by
+5. If all of the above is correctly set, login manually to your edge node and run your docker image manually by
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:14]:
```suggestion
6. If all of the above is correctly set, login manually to your edge node and run your docker image manually by
```

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
...
 
    ```shell
     docker run <your-container-image>
    ```
 
-7. If the docker container image is not pulled from the docker hub, please check that there is enough space on the edge node.
+6. If the docker container image is not pulled from the docker hub, please check that there is enough space on the edge node.
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:20]:
```suggestion
7. If the docker container image is not pulled from the docker hub, please check that there is enough space on the edge node.
```

File: docs/troubleshooting.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ This page contains a few commonly occurring questions.
 ## Container keeps pending/ terminating
 
 1. Check the output of `kubectl get nodes`
-2. Check the output of `kubectl describe pod <your-pod>`
+2. Use `kubectl describe pod <your-pod>` to check whether the pod is scheduled successfully.
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:8]:
```suggestion
2. Check the output of `kubectl describe pod <your-pod>`, whether the pod is scheduled successfully.
```

File: docs/troubleshooting.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ This page contains a few commonly occurring questions.
 ## Container keeps pending/ terminating
 
 1. Check the output of `kubectl get nodes`
```
Comment (kevin-wangzefeng) [inline docs/troubleshooting.md:7]:
```suggestion
1. Check the output of `kubectl get nodes`, whether the node is running healthy. Note that nodes in unreachable, offline status cannot complete graceful/soft pod deletion until they come back to normal.
```

**Pull request merged**